Python-isEven/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Autolab - autograding docker image
+
```

Comment:
It would be good to add an instruction here on how to build the Dockerfile, e.g. `docker build -t autograding_iseven .`